release_notes/ocp-4-11-release-notes.adoc

```diff
@@ -2344,9 +2344,7 @@ This script removes unauthenticated subjects from the following cluster role bin
 // TODO: This known issue should carry forward to 4.9 and beyond!
 * The `oc annotate` command does not work for LDAP group names that contain an equal sign (`=`), because the command uses the equal sign as a delimiter between the annotation name and value. As a workaround, use `oc patch` or `oc edit` to add the annotation. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1917280[*BZ#1917280*])
 
-* In the monitoring stack, if you have enabled and deployed a dedicated Alertmanager instance for user-defined alerts, you cannot silence alerts in the *Developer* perspective in the {product-title} web console.
-No workaround exists for this issue.
-(link:https://bugzilla.redhat.com/show_bug.cgi?id=2100860[*BZ#2100860*])
+* In the monitoring stack, if you have enabled and deployed a dedicated Alertmanager instance for user-defined alerts, you cannot silence alerts in the *Developer* perspective in the {product-title} web console. This issue has been fixed in 4.11.8. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2100860[*BZ#2100860*])
```

Comment:
Wanted to ask if there's a reason why "Developer" in developer perspective is bold and capitalized? I have not seen it as such elsewhere in docs, but if it is apologize! :)

Reply:
Got ya I see now! Thanks!